Backpack/src/main/res/drawable-nodpi/bpk_self_service.xml

```diff
@@ -1,3 +0,0 @@
-<vector xmlns:android="http://schemas.android.com/apk/res/android" android:width="24dp" android:height="24dp" android:viewportWidth="24" android:viewportHeight="24">
```

Comment:
this was somehow duplicated in another package and therefore overriding the original icon